recipes-browser/wpewebkit/wpewebkit_2.26.0.bb

```diff
@@ -0,0 +1,9 @@
+require wpewebkit.inc
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+SRC_URI = "https://wpewebkit.org/releases/${PN}-${PV}.tar.xz"
+SRC_URI[md5sum] = "59a2e64f145b3445187a22eff1f73293"
+SRC_URI[sha256sum] = "a97a24fe0c22243c873c359d7636512c01016ea2068a457ba651040f1a07ee3e"
+
+DEPENDS += " libwpe"
```

Comment:
This should move to the .inc file at some point :)

Reply:
i think this is there because we have still recipes for webkit using `wpebackend` not `libwpe`